editor/src/components/editor/global-shortcuts.tsx

```diff
@@ -806,7 +808,19 @@ export function handleKeyDown(
         }
         return [
           EditorActions.applyCommandsAction(
-            editor.selectedViews.flatMap((view) => nukeAllAbsolutePositioningPropsCommands(view)),
+            editor.selectedViews.flatMap((elementPath) => {
+              if (
+                MetadataUtils.isPositionAbsolute(
+                  MetadataUtils.findElementByElementPath(editor.jsxMetadata, elementPath),
+                )
+              ) {
+                return nukeAllAbsolutePositioningPropsCommands(elementPath)
+              }
+              return [
+                ...sizeToVisualDimensions(editor.jsxMetadata, elementPath),
+                ...positionAbsoluteRelativeToParentCommands(editor.jsxMetadata, elementPath),
+              ]
```

Comment:
I think this would read better as if-else, because these describe the two equal branches of the toggling behavior

Reply:
great idea, I added the explicit `else` branch in [b16532d](https://github.com/concrete-utopia/utopia/pull/3203/commits/b16532df3522ed46ef65d6438b2de19648b2fc9d)